tests/test_common.py

```diff
@@ -3,6 +3,72 @@
 import audformat
 
 
+@pytest.mark.parametrize(
+    'path, expected',
+    [
+        ('/root/file.txt', False),
+        ('./file.txt', False),
+        ('../file.txt', False),
+        ('doc/./file.txt', False),
+        ('doc/../file.txt', False),
+        ('doc/../../file.txt', False),
+        (r'C:\\doc\file.txt', False),
+        (r'a\b', False),
+        ('a/b', True),
+        ('b', True),
+        ('file.txt', True),
+    ],
+)
+def test_is_relative_path(path, expected):
+    relative = audformat.core.common.is_relative_path(path)
+    assert relative == expected
+
+
+def test_items_order():
+
+    d = audformat.core.common.HeaderDict(sort_by_key=False)
+    d['b'] = 1
+    d['c'] = 2
+    d['a'] = 0
+    assert dict(d.items()) == {'b': 1, 'c': 2, 'a': 0}
+    assert list(d.keys()) == ['b', 'c', 'a']
+    assert list(d.values()) == [1, 2, 0]
+    assert list(d) == ['b', 'c', 'a']
+    for item, expected in zip(d, ['b', 'c', 'a']):
+        assert item == expected
+    for item, expected in zip(reversed(d), ['a', 'c', 'b']):
+        assert item == expected
+    assert d.popitem(last=True) == ('a', 0)
+    assert d.popitem(last=False) == ('b', 1)
+    assert list(d) == ['c']
+
+    d = audformat.core.common.HeaderDict(sort_by_key=True)
+    d['b'] = 1
+    d['c'] = 2
+    d['a'] = 0
+    assert dict(d.items()) == {'a': 0, 'b': 1, 'c': 2}
+    assert list(d.keys()) == ['a', 'b', 'c']
+    assert list(d.values()) == [0, 1, 2]
+    assert list(d) == ['a', 'b', 'c']
+    for item, expected in zip(d, ['a', 'b', 'c']):
+        assert item == expected
+    for item, expected in zip(reversed(d), ['c', 'b', 'a']):
+        assert item == expected
+    assert d.popitem(last=True) == ('c', 2)
+    assert d.popitem(last=False) == ('a', 0)
+    assert list(d) == ['b']
+
+
+def test_meta_dict():
+    meta = {
+        'key': 'value',
+        'meta': 1234,
+    }
+    header = audformat.core.common.HeaderBase(meta=meta)
+    d = header.to_dict()
+    header.from_dict(d)
```

Comment:
... and here as well

```suggestion
    header.from_dict(d)
    assert header.meta['meta'] == 1234
```

Reply:
I locally updated the test